resources/v2/03_students_subjects.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE students_subjects
+(
+    id serial NOT NULL PRIMARY KEY,
+    student_id integer NOT NULL,
+    subject_id integer NOT NULL,
+    created_at timestamp
+    WITH time zone DEFAULT NOW(),
+    updated_at timestamp
+    WITH time zone,
+    deleted_at timestamp
+    WITH time zone,
+    CONSTRAINT students_subjects_student_id FOREIGN KEY (student_id) REFERENCES students(id),
+    CONSTRAINT students_subjects_subject_id FOREIGN KEY (subject_id) REFERENCES subjects(id)
+);
+
```

Comment:
also need to index the student_id and subject_id. Because you might required it later to search on it. Indexing will help you to search and respond faster

Reply:
We need to also add a unique constraint on subject_id and student_id, so duplicates dont come up

student_id, subject__id
2,2 -> valid
2,2 -> invalid (here it is duplicated so you need to unique on a combination of student_id, subject_id

https://www.w3schools.com/sql/sql_unique.asp

Reply:
Updated accordingly